lib/Transforms/IfTransform/IfTransform.td

```diff
@@ -0,0 +1,15 @@
+#ifndef LIB_TRANSFORMS_IFTRANSFORM_IFTRANSFORM_TD_
+#define LIB_TRANSFORMS_IFTRANSFORM_IFTRANSFORM_TD_
+
+include "mlir/Pass/PassBase.td"
+
+def IfTransform : Pass<"if-transform"> {
```

Comment:
Could we name this pass to better describe what it does? Perhaps `convert-if-to-select` like the pattern name.

Reply:
Great timing, I was just discussing this with Meron in our meeting while you posted this! We will change it to fit MLIR naming convetions :)